index.js

```diff
@@ -84,7 +84,25 @@ function makeDispatcher (fn) {
 module.exports.setGlobalDispatcher = setGlobalDispatcher
 module.exports.getGlobalDispatcher = getGlobalDispatcher
 
-module.exports.fetch = makeDispatcher(api.fetch)
+if (api.fetch) {
+  const _fetch = makeDispatcher(api.fetch)
+  module.exports.fetch = async function fetch (...args) {
+    try {
+      return await _fetch(...args)
```

Comment:
`return await` is useless if `async function`, you can just do `return _fetch(...args)` :)

Reply:
You forget the try/catch

Reply:
Ah indeed, you're right